### DISCUSSION TITLE
Components Disappeared but Show in List

### DISCUSSION THREAD

**Discussion by mmalessa80** [Budibase/budibase#7248] [Help] [answered]
One of my pages lists my components in the left hand list, however they don't show up on the page. If I preview the app and navigate to that page they don't show up there either.

![Screenshot 2022-08-12 9 00 30 AM](https://user-images.githubusercontent.com/106758851/184384126-1c6cda0b-4902-4597-9c51-395e2b206204.png)

**Comment by andz-bb** [comment, marked as answer]
Hey, have you seen any errors in the console or network tab (F12 in Chrome) that would suggest that something isn't loading in properly? Is it the same when you refresh the page?

**Reply by mmalessa80** [reply]
This what the console looks like:

![image](https://user-images.githubusercontent.com/106758851/184388929-099c6e5f-de8b-4bca-b6c8-a9653ef7f23b.png)

Running the latest version and had just updated prior to looking at it. It has been a while since I worked on the app.

**Reply by andz-bb** [reply]
Thanks for showing me that, would you be able to take an export of the app so that I can have a closer look on my end? If you send it to support@budibase.com I can investigate further. Cheers

**Reply by mmalessa80** [reply]
I have sent my app export to the indicated email address. Please note that in design view I can't see any of my components on any of my screens now. That seems to be different from Friday when I first noted this. When doing a preview I can access all screens apparently with exception of screen /annualpermitdata-2/new/row.

**Reply by andz-bb** [reply]
Thanks, I've got the app export here and will investigate it for you soon

**Reply by mmalessa80** [reply]
After looking into this a little bit more I noticed the following:

While in design view I can select any screen from the screens list on the left hand or from the screens dropdown and the components will show. Once I select the /annualpermitdata-2/new/row screen the components on that screen will not show in design. Any subsequent screen selection after that will no longer show the components on the other screens.

**Reply by andz-bb** [reply]
I've played around with your app and I think I have found the problem. On the /annualpermitdata-2/new/row screen, there is a component called New Button with Amount Used as the text. It has one condition set shown here:
![amount used button condition](https://user-images.githubusercontent.com/110921612/184684497-fc6e4fda-fa74-4463-ae3e-eaa1e6774778.png)

I think this is causing the front end to break, specifically the `Is In` condition. If you change `Is In` to `Equals`, save and then refresh you should find that the screen renders properly without erroring.

![using equals it renders ok](https://user-images.githubusercontent.com/110921612/184684850-a2d45f43-7995-4f44-a8bb-ce4182238d08.png)

I will have a look tomorrow to see what we can do to make this configuration safer in future. Hope this helps you, let me know if it fixes your app! 😊

**Reply by andz-bb** [reply]
@mmalessa80 just to follow up, I have created an issue from this and it should be fixed soon 👍 https://github.com/Budibase/budibase/issues/7296
